pages_builder/pages/forms/checkbox-tree.yml

```diff
@@ -0,0 +1,176 @@
+pageTitle: Checkbox tree
+assetPath: ../govuk_template/assets/
+grid: column-two-thirds
+bodyEnd: >
+  <script type="text/javascript" src="../public/javascripts/govuk_frontend_toolkit/vendor/polyfills/bind.js"></script>
+  <script type="text/javascript" src="../public/javascripts/govuk_frontend_toolkit/govuk/selection-buttons.js"></script>
+examples:
+  - |
+    <form>
+
+    {%
+      with
+      name="serviceCategories-10",
+      question="Which categories does your service fit under?",
+      question_advice="",
+      hint="You can choose multiple categories.",
+      id="serviceCategories-10",
+      options=[
+        {"label": "Archiving, backup and disaster recovery"},
```

Comment:
Yeah, I'm just using the real categories we're using. 
It's not fun, but it was easy.